src/flavors/duwamish_flavor/jstemplates/place-form.html

```diff
@@ -0,0 +1,91 @@
+
```

Comment:
This looks great, thanks. But can you try breaking your commits into smaller, logical chunks? For example, it is hard for me to visualize what has changed in this file. If you uploaded the file, commit it, change the file, and commit again, then I can see what has been changed.